packages/notifi-frontend-client/lib/client/clientFactory.ts

```diff
@@ -42,3 +43,9 @@ export const newEvmClient = (config: NotifiEvmConfiguration) => {
   const storage = newNotifiStorage(config);
   return new NotifiFrontendClient(config, service, storage);
 };
```

Comment:
for these client factories,
is possible to have a type that inclues something like this?

type NotifiConfiguration =
  | NotifiAptosConfiguration
  | NotifiInjectiveConfiguration
  | NotifiSolanaConfiguration
  | NotifiEvmConfiguration;


and then try to generate a new frontendClient?

Reply:
Nice point, will implement it in MVP-2537